openapi.yaml

```diff
@@ -16022,6 +16041,38 @@ components:
           minimum: 1
           maximum: 100
           example: 6
+        disks:
+          type: array
+          description: >
+            This Node Pool's custom disk layout. Each item in this array will create a new disk
+            partition for each node in this Node Pool.
+
+              * The custom disk layout is applied to each node in this Node Pool.
+              * The maximum number of custom disk partitions that can be configured is 7.
+              * A Node Pool's custom disk layout is immutable over the lifetime of the Node Pool.
+          items:
+            type: object
+            description: >
+              The values to assign to each partition in this Node Pool's custom disk layout.
+            properties:
+              size:
+                description: >
+                  The size of this custom disk partition in MB. The size of this disk partition must not exceed the
```

Comment:
I wouldn't bother noting the sum of sizes validation, the API validation errors will make it clear. it's a bit complex to describe accurately, since there's a minimum necessary slice we reserve for the boot disk.

Reply:
thinking about this more; it's worth noting somewhere that whatever disk is leftover once the requested disks are allocated is allocated to the boot disk

Reply:
Added note about boot disk allocation: https://github.com/linode/linode-api-docs/pull/283/commits/347c6a18186f0f7d42894177e55d08383d08166f#diff-fe030a7c1568b3decf599edf399be7f4R16052